install/kubernetes/istio-auth.yaml

```diff
@@ -554,21 +554,62 @@ data:
   mesh: |-
     # Uncomment the following line to enable mutual TLS between proxies
     authPolicy: MUTUAL_TLS
+    #
+    # Set the following variable to true to disable policy checks by the Mixer.
+    # Note that metrics will still be reported to the Mixer.
+    disablePolicyChecks: false
+    # Set enableTracing to false to disable request tracing.
+    enableTracing: true
+    #
+    # To disable the mixer completely (including metrics), comment out
+    # the following line
+    mixerAddress: istio-mixer.istio-system:9091
+    #
     ingressService: istio-ingress.istio-system
     egressProxyAddress: istio-egress.istio-system:80
-    mixerAddress: istio-mixer.istio-system:9091
+    #
+    # Along with discoveryRefreshDelay, this setting determines how
+    # frequently should Envoy fetch and update its internal configuration
+    # from Istio Pilot. Lower refresh delay results in higher CPU
+    # utilization and potential performance loss in exchange for faster
+    # convergence. Tweak this value according to your setup.
+    rdsRefreshDelay: 30s
+    #
     defaultConfig:
-          configPath:             "/etc/istio/proxy"
-          binaryPath:             "/usr/local/bin/envoy"
-          serviceCluster:         istio-proxy
-          drainDuration:          2s
-          parentShutdownDuration: 3s
-          discoveryAddress:       istio-pilot.istio-system:8080
-          discoveryRefreshDelay:  1s
-          connectTimeout:         1s
-          proxyAdminPort:         15000
-          zipkinAddress:          zipkin.istio-system:9411
-          statsdUdpAddress: istio-mixer.istio-system:9125
+      # See rdsRefreshDelay for explanation about this setting.
+      discoveryRefreshDelay: 30s
```

Comment:
Did you intentionally remove the indent under defaultConfig?  Seems it reads better with the indent.

Reply:
This seems to be the indent that emacs gave me in YAML mode.